src/js/view/controls/settings-menu.js

```diff
@@ -85,12 +85,16 @@ export function setupSubmenuListeners(settingsMenu, controlbar, viewModel, api)
             return;
         }
 
+        const localization = model.get('localization');
+
         addQualitiesSubmenu(
             settingsMenu,
             levels,
             (index) => api.setCurrentQuality(index),
             model.get('currentLevel'),
-            model.get('localization').hd
+            localization.hd,
+            localization.auto,
+            localization.quality
```

Comment:
seems like `localization.quality` is not being used; is it deprecated ?

Reply:
I didn't know we had localization.quality. I thought it was only localization.hd. The default for that is "Quality"